index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Alzubair Protfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/de9dd03ce2.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="header">
+    <header class="container">
+      <h1 class="navbar">My Logo</h1>
```

Comment:
- [ ] Please the logo has a link `<a>` tag instead of `<h1>` tag. This is because the logo for the company is similar to the name of the company. Therefore it would make sense to hyperlink a key person's name or a company name. This makes your job more professional.

Reply:
hello @OLIPLICHE  I have done everything can you check it again please